1.1.c

```diff
@@ -0,0 +1,100 @@
+#include <string.h>
+#include <stdio.h>
+
+#define binarySize 256
+
+
+int transformation2ndWay(char* binary, char* octale)	//https://calcus.ru/perevod-sistem-schisleniya/iz-dvoichnoy-v-vosmerichnuyu here i took a way to transformation (2nd way)
+{
+int result = 0;

```

Comment:
```suggestion
    int transformedBlock = 0;
```
1) Форматирование отступов в теле функции: все строки в этой функции должны быть с отступом (-0.25)
2) Должно быть более информативное имя. (-0.25)